index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en"> 
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta property="og:image" content="./desktop_screenshot.png">
+  <title>Golden View School</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/f8b4998ade.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="pre-nav">
+    <a href=""> <i class="fab fa-twitter"></i></a>
+    <a href=""><i class="fab fa-instagram"></i></a>
+    <ul class="pre-menu">
+      <li><a href="#">English</a></li>
+      <li><a href="#">My Page</a></li>
+      <li><a href="#">Logout</a></li>
+    </ul>
+  </div>
+  <nav id="nav" class="invisible">
+    <div id="cross" class="cross-container" onclick="toggleMenu()">
+      <i class="fas fa-times"></i>
+    </div>
+    <a href="./index.html" class="nav-logo-frame">
+      <img src="./assets/Golden-view.jpg" alt="" class="nav-logo">
+    </a>
+    <ul class="menu">
+      <li><a href="./about.html">About</a></li>
+      <li><a href="#" onclick="toggleMenu()">Program</a></li>
+      <li><a href="#" onclick="toggleMenu()">Join</a></li>
+      <li><a href="#" onclick="toggleMenu()">Sponsor</a></li>
+      <li><a href="#" onclick="toggleMenu()">News</a></li>
+    </ul>
+    <button type="button" class="nav-btn">Join Now!</button>
+    <div class="deco-ball-1"></div>
+    <img src="./assets/11-chess-board-gray.png" alt="" class="curtain-deco">
+  </nav>
+  <div class="img-frame">
+    <div id="burger-container" class="burger-container">
+      <i class="fas fa-bars"></i>
+    </div>
+    <img src="./assets/nairobi.jpg" alt="View of Nairobi City" class="back-img">
+    <div class="img-curtain">
+      <img src="./assets/11-chess-board-gray-resized.png" alt="" class="curtain-deco">
+      <img src="./assets/hitmark-gray.png" alt="" class="curtain-hitmark">
+    </div>
+  </div>
+  <header>
+    <h1>Hey There</h1>
+    <h2>Golden View <br>SChool<br>Welcomes you</h2>
+    <div class="deco-ball-1"></div>
+    <div class="txt-container">
```

Comment:
- [ OPTIONAL] Kindly try to give a margin to your elements correctly to match the original design 

![image](https://user-images.githubusercontent.com/98453979/196746211-55024da5-325b-4423-8b3f-17eb56cd8848.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/196746476-a3a60185-206d-4c2c-9a59-c2aa4c31335c.png)

- [OPTIONAL] Kindly provide a margin top and bottom of this section to match the original design

![image](https://user-images.githubusercontent.com/98453979/196747027-c817436c-6b92-4cbd-ac83-3c80ed3cdb45.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/196747208-09c91233-05a3-405d-9fb5-a85b72ca739f.png)

- [OPTIONAL] Kindly enable this hover effect on your cards (white borderline)  to match the original design

![image](https://user-images.githubusercontent.com/98453979/196747723-353ba494-2b9f-4682-905a-843b43737306.png)